### PR TITLE
move the localStorage lookup out of the needs declaration

### PR DESCRIPTION
**Pull request by michaeljb**
addresses https://github.com/tobymao/18xx/pull/2409#discussion_r528455233

### CODE REVIEW

File: assets/app/view/game/map.rb [hunk 1]
```diff
@@ -174,6 +174,22 @@ def render_map
           ]),
         ])
       end
+
+      def show_coords
+        default = false
+        stored = Lib::Storage['show_coords']
+        default = stored.nil? ? default : stored
```
Comment (tobymao) [inline assets/app/view/game/map.rb:181]: default = stored.nil? ? false : Lib::Storage['show_coords']

Comment (ryandriskel) [reply assets/app/view/game/map.rb:181]: toby, Why access Lib twice? ( the first time defines stored )

Comment (ryandriskel) [reply assets/app/view/game/map.rb:181]: `def show_coords
        stored = Lib::Storage['show_coords']
        default = stored.nil? ? false : stored`

Seems like maybe what you meant?

Comment (tobymao) [reply assets/app/view/game/map.rb:181]: yes thanks

**Pull request merged**